charts/netbox/templates/NOTES.txt

```diff
@@ -2,31 +2,51 @@ CHART NAME: {{ .Chart.Name  }}
 CHART VERSION: {{ .Chart.Version  }}
 APP VERSION: {{ .Chart.AppVersion  }}
 
+** Please be patient while the chart is being deployed **
+
+Netbox can be accessed through the following DNS name from within the cluster:
+
+    {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} (port {{ .Values.service.port }})
+
+To access Netbox site from outside the cluster follow the steps below.
+
 {{- if .Values.ingress.enabled }}
-NetBox should be available at the following URL(s) shortly:
+
```

Comment:
Since the bit above now says "follow the steps below" this should probably still include a sentence of some kind and not just URLs.

Maybe something like:

> You have configured NetBox to use an ingress controller, exposing the following URL(s):

?

Reply:
Oh indeed, I forgot to put it back, good catch!
Fixed